gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
-kotlin = "1.8.21"
-ksp = "1.8.21-1.0.11" # depends on Kotlin version
-composeCompiler = "1.4.7" # depends on Kotlin version
+kotlin = "1.9.0"
+ksp = "1.9.0-1.0.12" # depends on Kotlin version
+composeCompiler = "1.5.0" # depends on Kotlin version
 compilerTesting = "1.5.0"
 destinations = "1.8.42-beta" # update this alongside with compiler version
```

Comment:
We can use 1.9 (which is stable, btw), but it brings the latest compose beta libraries which we don't want for now. They also require compileSdk=34, which we don't want as well for now